docs/technical_reference/costing/WT_unit_costing_for_docs.csv

```diff
@@ -0,0 +1,27 @@
+﻿name,unit
```

Comment:
I could probably get rid of this csv and just read filenames from costing/unit_models, but isn't _necessary_ to make that change yet.